src/test/java/org/elasticsearch/indices/analyze/AnalyzeActionTests.java

```diff
@@ -191,4 +198,76 @@ private void assertTokens(AnalyzeResponse response, String ... tokens) {
     private static String indexOrAlias() {
         return randomBoolean() ? "test" : "alias";
     }
+
+    @Test
+    public void testParseAnalyzeRequest() throws Exception {
```

Comment:
When the parsing logic is on the transport layer then we don't need these tests, because it is covered by the other tests?